homeassistant/components/saj/config_flow.py

```diff
@@ -0,0 +1,125 @@
+"""Config flow for saj integration."""
+import logging
+
+import pysaj
+import voluptuous as vol
+
+from homeassistant import config_entries, data_entry_flow
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.const import (
+    CONF_DEVICE_ID,
+    CONF_HOST,
+    CONF_NAME,
+    CONF_PASSWORD,
+    CONF_SCAN_INTERVAL,
+    CONF_TYPE,
+    CONF_USERNAME,
+)
+from homeassistant.core import callback
+
+from .const import (  # pylint:disable=unused-import
```

Comment:
Should not be needed to disable pylint here

```suggestion
from .const import (
```